examples/ssr-caching/server.js

```diff
@@ -10,8 +10,8 @@ const handle = app.getRequestHandler()
 
 const ssrCache = cacheableResponse({
   ttl: 1000 * 60 * 60, // 1hour
-  get: async ({ req, res, pagePath, queryParams }) => {
-    const data = await app.renderToHTML(req, res, pagePath, queryParams)
+  get: async ({ req, res }) => {
+    const data = await app.renderToHTML(req, res, req.path, req.param)
```

Comment:
👋 @xliusstk! Should it be `req.query` instead of `req.param`?

Reply:
Yep @kachkaev I've fixed it 🙏